packages/big-design/src/components/Collapse/Collapse.tsx

```diff
@@ -0,0 +1,63 @@
+import { ExpandMoreIcon } from '@bigcommerce/big-design-icons';
+import React, { useState } from 'react';
+
+import { useUniqueId } from '../../hooks';
+import { typedMemo } from '../../utils';
+import { Box } from '../Box';
+
+import { StyledButton } from './styled';
+
+export interface CollapseProps extends React.HTMLAttributes<HTMLDivElement> {
+  title?: string;
+  initiallyOpen?: boolean;
+  onCollapseChange?(isOpen: boolean): void;
+}
+
+export const Collapse: React.FC<CollapseProps> = typedMemo(
+  ({ children, title, onCollapseChange, initiallyOpen = false }) => {
+    const [isOpen, setIsOpen] = useState(initiallyOpen);
+    const triggerId = useUniqueId('collapse-title');
+    const panelId = useUniqueId('collapse-panel');
+
+    const handleTitleClick = () => {
+      const nextIsOpen = !isOpen;
+
+      setIsOpen(nextIsOpen);
+
+      if (typeof onCollapseChange === 'function') {
+        onCollapseChange(nextIsOpen);
+      }
+    };
+
+    if (!title) {
```

Comment:
Let's make `title` a required prop and get rid of this assignment.

Reply:
It was discussed with the designer last week that. Default title should change from "Show more" to "Show less" and vice versa.

There are two ways to do it:
1. Always change title using `onCollapseChange` and make title prop required.
2. Set default title values in collapsed and expanded states.

Changed for version without default title. Now we should always provide title and in most cases manage title change using `onCollapseChange`. On the Collapse example page I set default values which were discussed.

Reply:
That should be something we push back on. These components should be kind of "dumb" in one sense, by letting the app dictate what the title is. We also want to avoid adding our own text into the component library since we haven't figured out i18n yet.

One more change you need to make is on line 10 is to remove the `?` from the `title` type, like so:
```ts
 title: string;
```